ivy/__init__.py

```diff
@@ -312,18 +318,26 @@ class Node(str):
     float32,
     float64,
 )
+all_uint_dtypes = (
```

Comment:
now that we have these added, we should also add them to each of the backend `__init__.py` files as well (for example the torch [`__init__.py`](https://github.com/unifyai/ivy/blob/591ac37a664ebdf2ca50a5b0751a3a54ee9d5934/ivy/functional/backends/torch/__init__.py) file).

You can follow the same format as for the others, such as `valid_int_dtypes`, `valid_float_dtypes` etc.